src/main/java/com/google/sps/servlets/AnalyzedThread.java

```diff
@@ -0,0 +1,96 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Random;
+
+public class AnalyzedThread {
+  public static String[] titlesArray = {"Where is this supposed to go?",
+      "This program needs to work better", "The wifi needs to be more consistent",
+      "This could be better", "Maybe this could be structured a different way"};
+  public static double[] sentimentArray = {-.3, -.5, -.1, -.2, -.5};
+  public static int[] upvotesArray = {30, 50, 10, 5, 12};
+  public static String[] urlArray = {"http://www.youtube.com", "http://www.reddit.com",
+      "http://www.facebook.com", "http://www.yahoo.com", "http://www.amazon.com"};
+  private static String threadTitle;
+  private static double sentiment;
+  private static int upvotes;
+  private static int timestamp;
+  private static String threadUrl;
```

Comment:
These don't need to be static, what do you think of removing the `static` keyword?